CHANGELOG.md

```diff
@@ -66,6 +66,7 @@ Since 2018.1, the version numbers and release cycle match Rider's versions and r
 - Fix asset usages not working with partial classes ([RIDER-50125](https://youtrack.jetbrains.com/issue/RIDER-50125), [#1935](https://github.com/JetBrains/resharper-unity/pull/1935))
 - Fix exceptions when running plugin as part of ReSharper command line tools ([RIDER-53968](https://youtrack.jetbrains.com/issue/RIDER-53968), [#1935](https://github.com/JetBrains/resharper-unity/pull/1935))
 - Fix false positive reports of repeated property access warnings with `MonoBehaviour` based components ([#1544](https://github.com/JetBrains/resharper-unity/issues/1544), [#1935](https://github.com/JetBrains/resharper-unity/pull/1935))
+- Fix profiling Unity from Rider should not force DebugCodeOptimization ([RIDER-54917](https://youtrack.jetbrains.com/issue/RIDER-54917), [#1954](https://github.com/JetBrains/resharper-unity/pull/1954))
```

Comment:
Start this with `Rider: ` or it will appear in the ReSharper release notes

Reply:
got it, thanks!